build/pxc-entrypoint.sh

```diff
@@ -488,6 +491,56 @@ if [ "$1" = 'mysqld' -a -z "$wantHelp" ]; then
 		fi
 		rm "$wsrep_verbose_logfile"
 	fi
+	if [ -n "$PXC_SERVICE" ]; then
+		function get_primary() {
+			peer-list -on-start=/usr/bin/get-pxc-state -service="$PXC_SERVICE" 2>&1 \
+				| grep wsrep_ready:ON:wsrep_connected:ON:wsrep_local_state_comment:Synced:wsrep_cluster_status:Primary \
+				| sort \
+				| tail -1 \
+				|| true
+		}
+		function node_recovery() {
+			set -o xtrace
+			echo "Recovery is in progress, please wait...."
+			sed -i 's/wsrep_cluster_address=.*/wsrep_cluster_address=gcomm:\/\//g' /etc/mysql/node.cnf
+			rm -f /tmp/recovery-case
+			if [ -s "$grastate_loc" ]; then
+				sed -i 's/safe_to_bootstrap: 0/safe_to_bootstrap: 1/g' "$grastate_loc"
+			fi
+			echo "Recovery was finished."
+			exec "$@" $wsrep_start_position_opt
+		}
+
+		is_primary_exists=$(get_primary)
+		if [[ -z "$is_primary_exists" && -f "$grastate_loc" && $safe_to_bootstrap != 1 ]] || [[ -z "$is_primary_exists" && -f "${DATADIR}/gvwstate.dat" ]]; then
+			trap "{ node_recovery \"\$@\" ; }" USR1
+			touch /tmp/recovery-case
+			if [[ -z ${seqno} ]]; then
+				seqno="-1"
+			fi
+
+			set +o xtrace
+			sleep 3
+
+			echo '#####################################################FULL_PXC_CLUSTER_CRASH#####################################################'
```

Comment:
```suggestion
			echo "#####################################################FULL_PXC_CLUSTER_CRASH:$NODE_NAME#####################################################"
```

I think in this case we will have more beautiful message.